runtime/src/main/java/org/corfudb/runtime/CorfuRuntime.java

```diff
@@ -533,6 +526,9 @@
     @Getter
     public volatile UUID clusterId;
 
+    @Getter
+    public volatile OrderedGUIDGenerator guidGenerator;
```

Comment:
you don't need public if you use `@Getter`

Reply:
it can be private final

Reply:
good point. fixed. thanks!